lib/foreman_discovery/engine.rb

```diff
@@ -88,7 +89,8 @@ class Engine < ::Rails::Engine
               setting "discovery_prefix",
                 type: :string,
                 default: "mac",
-                validate: { presence: true, format: { with: /\A[a-zA-Z]/, message: _("must start with a letter") } },
+                validate: { presence: true, format: { with: /\A([A-Za-z][A-Za-z0-9-]{0,61})\z/,
```

Comment:
For a smaller diff I'd suggest
```suggestion
                validate: { presence: true, format: { with: /\A[a-zA-Z][a-zA-Z0-9-]{0,61}\z/,
```

Reply:
@stejskalleos this is marked as resolved, while it isn't.